src/main/java/in/koreatech/koin/domain/graduation/model/GradeExcelData.java

```diff
@@ -15,29 +16,32 @@ public record GradeExcelData(
     String grade,
     String retakeStatus
 ) {
-    public static GradeExcelData fromRow(Row row) {
-        return new GradeExcelData(
-            getCellValueAsString(row.getCell(1)),
-            getCellValueAsString(row.getCell(2)),
-            getCellValueAsString(row.getCell(4)),
-            getCellValueAsString(row.getCell(5)),
-            getCellValueAsString(row.getCell(6)),
-            getCellValueAsString(row.getCell(7)),
-            getCellValueAsString(row.getCell(8)),
-            getCellValueAsString(row.getCell(9)),
-            getCellValueAsString(row.getCell(10)),
-            getCellValueAsString(row.getCell(11))
-        );
+
+    private static final String MIDDLE_TOTAL = "소 계";
+    private static final String FAIL = "F";
+    private static final String UNSATISFACTORY = "U";
+    private static final String TOTAL = "합 계";
+    private static final String FIRST_SEMESTER = "1";
+    private static final String SECOND_SEMESTER = "2";
+    private static final String SUMMER_SEMESTER = "동계";
+    private static final String WINER_SEMESTER = "여름";
```

Comment:
오타난 것 같은데 확인 부탁드릴게요!!
```suggestion
    private static final String SUMMER_SEMESTER = "하계";
    private static final String WINTER_SEMESTER = "동계";
```

Reply:
헉 오타가 있었네요 !

[120e61e](https://github.com/BCSDLab/KOIN_API_V2/pull/2045/commits/120e61e8edc4954ec4638bbb648ffdcd87345164)에서 수정했습니다 ~